Recent changes now require C99 (build breaks on VxWorks 6.9)

I see these compile errors on VxWorks 6.9:
```
/ics/vxWorks/6.9/WindRiver/gnu/4.3.3-vxworks-6.9/x86-linux2/bin/ccppc           -DSUPPORT_LINT  -DCPU=PPC604  -DvxWorks=vxWorks -fdollars-in-identifiers -include /ics/vxWorks/6.9/WindRiver/vxworks-6.9/target/h/vxWorks.h     -O2   -Wall -Werror-implicit-function-declaration     -DUSE_TYPED_RSET -DUSE_TYPED_DSET -DUSE_TYPED_DRVET -DSUPPORT_LINT  -mcpu=604 -mstrict-align -mlongcall -DMV5500    -fno-builtin -fexceptions -fno-strict-aliasing   -I. -I../O.Common -I. -I. -I.. -I../../../include/compiler/gcc -I../../../include/os/vxWorks -I../../../include    -I../../../../../base/include/compiler/gcc -I../../../../../base/include/os/vxWorks -I../../../../../base/include       -I/ics/vxWorks/6.9/WindRiver/vxworks-6.9/target/h -I/ics/vxWorks/6.9/WindRiver/vxworks-6.9/target/h/wrn/coreip -c ../ether_ip.c
../ether_ip.c: In function 'EIP_describe_type':
../ether_ip.c:2809: error: 'for' loop initial declaration used outside C99 mode
../ether_ip.c:2898: error: redefinition of 'i'
../ether_ip.c:2809: error: previous definition of 'i' was here
../ether_ip.c:2898: error: 'for' loop initial declaration used outside C99 mode
../ether_ip.c:2911: error: redefinition of 'i'
../ether_ip.c:2898: error: previous definition of 'i' was here
../ether_ip.c:2911: error: 'for' loop initial declaration used outside C99 mode
make[3]: *** [../../../../../base/configure/RULES_BUILD:260: ether_ip.o] Error 1
make[3]: Leaving directory '/ics/epics/7.0.10/ether_ip/fix-vxworks_build/ether_ipApp/src/O.vxWorks-6_9-mv5500'
make[2]: *** [../../../../base/configure/RULES_ARCHS:58: install.vxWorks-6_9-mv5500] Error 2
make[2]: Leaving directory '/ics/epics/7.0.10/ether_ip/fix-vxworks_build/ether_ipApp/src'
make[1]: *** [../../../base/configure/RULES_DIRS:85: src.install] Error 2
make[1]: Leaving directory '/ics/epics/7.0.10/ether_ip/fix-vxworks_build/ether_ipApp'
make: *** [../../base/configure/RULES_DIRS:85: ether_ipApp.install] Error 2
```

but enabling C99 fixes it:
```
diff --git a/ether_ipApp/src/Makefile b/ether_ipApp/src/Makefile
index a193b30..a7f299d 100644
--- a/ether_ipApp/src/Makefile
+++ b/ether_ipApp/src/Makefile
@@ -8,6 +8,8 @@ include $(TOP)/configure/CONFIG
 # On Darwin, got errors about unused assignment in recGblSetSevr
 USR_CFLAGS_Darwin += -Wno-unused-value
 
+USR_CFLAGS_vxWorks += -std=c99
+
 # On WIN32, have to add socket library
 # to build the test program
 SYS_PROD_LIBS_WIN32 = wsock32

```

Is this a reasonable fix? 

Although, it would be simple to change the code to pre-C99 standard if we want to retain that.

--- a/ether_ipApp/src/Makefile
+++ b/ether_ipApp/src/Makefile
@@ -12,6 +12,9 @@ USR_CFLAGS_Darwin += -Wno-unused-value
 # to build the test program
 SYS_PROD_LIBS_WIN32 = wsock32
 
+# Need to enable C99 for VxWorks
+USR_CFLAGS_vxWorks += -std=c99
+
 ifdef BASE_3_15
 devEtherIP_CFLAGS += -DBUILD_LONG_STRING_SUPPORT
 USR_CFLAGS += -DUSE_TYPED_RSET -DUSE_TYPED_DSET -DUSE_TYPED_DRVET

